docs/concepts-new/mesh.md

```diff
@@ -4,31 +4,55 @@ title: Introduction to Temporal's core concepts
 sidebar_label: Introduction
 ---
 
-:::note
+:::caution
 
 This page is a work in progress!
 
 :::
 
-**Welcome to Temporal's core terminology!**
+**Temporal's tenth rule**
+
+> Any sufficiently complex distributed system contains an ad-hoc, informally-specified, bug-ridden, slow implementation of half of temporal.io.
+
+## What is Temporal?
+
+[temporal.io](https://temporal.io) is a scalable and reliable runtime for Temporal Workflow Executions.
+
+<img class="docs-image-centered docs-image-max-width-50" src="/img/temporal-intro.png" />
+
+A [Temporal Workflow Execution](#what-is-a-workflow-execution) (π) executes a [Temporal Workflow Definition](#workflow-definition), also called a Temporal Workflow Function, your application code, exactly once and to completion—whether your code executes for seconds, minutes, hours, days, months, or years, in the presence of arbitrary load and arbitrary failures.
+
+A Temporal Application is a set of Temporal Workflow Executions (Π). Each Temporal Workflow Execution has exclusive access to its local state, executes concurrently to all other Workflow Executions and communicates with other Workflow Executions and the environment via message passing.
 
-This page is dedicated to defining and describing the terms, components, and concepts of the Temporal system in a holistic way.
+Workflow Executions are lightweight components, a Temporal Application may consist of thousands to hundreds of thousands of Workflow Executions. A Workflow Execution consumes few compute resources; in fact, if a Workflow Execution is suspended, for example if the Workflow Execution is in a waiting state, the Workflow Execution consumes no compute resources at all.
 
-## Workflow
+The temporal.io runtime
```

Comment:
I'm guessing on intent here. Feel free to stet or change.

```suggestion
The Temporal runtime consists of the Temporal Server and Temporal SDKs.
```

Reply:
Ya, obviously spaced out and didn't finish that :)